extensions/cornerstone/src/init.js

```diff
@@ -275,3 +287,228 @@ const _connectToolsToMeasurementService = measurementService => {
     }
   );
 };
+
+const _enableReferenceLines = () => {
+  const waitForImageRendered = enabledElement =>
```

Comment:
Is this your "null safety", but for making sure a cornerstoneElement has rendered an image?

Reply:
Exact. This value can be null If you choose a 2 x 1 viewport layout and change the image index of the first viewport before the image of the second viewport is rendered.